CMakeLists.txt

```diff
@@ -299,6 +299,10 @@ install(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/rpp PATTERN "*.h.in" EXCLUDE)
 install(FILES ${PROJECT_BINARY_DIR}/include/rpp_backend.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/rpp)
 
+enable_testing()
+include(CTest)
+add_subdirectory(utilities/rpp-unittests)
```

Comment:
We may need to just do this for the new test_suite folder with all the latest tests. The rpp-unittests and rpp-performancetests would go away after tensor code consolidation.

Reply:
When will the new test suite be ready?